confignode/src/main/java/org/apache/iotdb/confignode/manager/load/service/StatisticsService.java

```diff
@@ -65,12 +67,15 @@ public class StatisticsService implements IClusterStatusSubscriber {
   private final LoadCache loadCache;
   private final EventBus eventBus;
 
+  private final Object recordLock;
```

Comment:
What's the differentce if you add `synchronized` on function `onClusterStatisticsChanged` `onRegionLeaderChanged`?

Reply:
I want the `onClusterStatisticsChanged` and `onRegionLeaderChanged` in StatisticsService be serialized. So that the logs recorded are easier to read. Therefore I need a `recordLock` here.

Reply:
But it seems you can synchronized on function onClusterStatisticsChanged onRegionLeaderChanged directly to achieve your goal without introducing this field?

Reply:
Fixed